src/EventTimer.spec.ts

```diff
@@ -50,6 +57,56 @@ describe('EventTimer', () => {
 		EventTimer.init();
 	});
 
+	it('get correct cmp events', () => {
+		const performanceCMP = {
+			now: jest.fn(),
+			mark: jest.fn(),
+			getEntriesByName: mockGetEntriesByName([
+				'cmp-tcfv2-init',
+				'cmp-tcfv2-got-consent',
+			]),
+		};
+		Object.defineProperty(window, 'performance', {
+			configurable: true,
+			enumerable: true,
+			value: performanceCMP,
+			writable: true,
+		});
+		const eventTimer = EventTimer.get();
+		expect(eventTimer.events).toHaveLength(2);
+		expect(eventTimer.events.map((event) => event.name)).toEqual(
+			expect.arrayContaining(['cmp-tcfv2-init', 'cmp-tcfv2-init']),
```

Comment:
this array is different from the array above.

```suggestion
			expect.arrayContaining(['cmp-tcfv2-init', 'cmp-tcfv2-got-consent']),
```

You might want to extract this to a const inside the test function anyways? Keep it DRY?

Reply:
Fixed it, thanks!

Reply:
I’m sort of worried that the test passed in the first place… any idea why that was?

Reply:
We found out it's because `arrayContaining` checks for elements being present in the array *at least* once, so it doesn't throw an error if the same element appears twice. We got around the issue by sorting the expected and the actual arrays and checking for equality instead.